### ISSUE TITLE
New Tractus-X project contributor

### ISSUE THREAD

**Issue by alexandrudanciu** [eclipse-tractusx/sig-infra#494]
## Necessary Information

Email address to add: ciprian.nicuta@sap.com
Committers, that could take over: @arnoweiss

**Comment by arnoweiss** [comment]
I'm slightly lost here. 

I understand that the purpose to add the relevant accounts as contributors is to assign tickets to them. Making someone a contributor is possible via this github-issue template OR according to the [handbook](https://www.eclipse.org/projects/handbook/#contributing-contributors) (i.e. get a PR merged). Even if there was a process to bypass the contribution to become a contributor, as committer, I wouldn't know how to.

**Comment by SebastianBezold** [comment]
HI @arnoweiss,

I wouldn't say, assigning tickets is the "purpose" of inviting someone to be a Tractus-X project contributor.
But doing so, the invited person will be granted "triage" permissions.
This issue here and the relate process was introduced, because in the beginning, no one really knew how to do that (plus, we wanted to keep track of new contributors). 
But since we are becoming more and more committers, the task can be shared across all of them.

So if you are not sure, how to add someone to the project contributor list, I am happy to show it to you. 
Basically it is done via the Project Managment Interface (PMI): https://projects.eclipse.org/projects/automotive.tractusx
If you are logged in as committer, you will see an "Edit" tab. 
It seems like this does not work right now, but I'm sure, the EF will fix it soon

**Comment by arnoweiss** [comment]
Thanks! Yea, still not seeing the "edit" tab. Any idea where to file a ticket for this?

**Comment by SebastianBezold** [comment]
Hi @arnoweiss,

if you are logged in and have committer permissions, you should at least see the button.
But it is a confirmed issue now, that it is not working: https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/4713. 
The Eclipse Foundation is working on it

**Comment by FaGru3n** [comment]
FYI https://gitlab.eclipse.org/eclipsefdn/it/websites/projects.eclipse.org/-/issues/323

**Comment by FaGru3n** [comment]
Hi all, 

fyi "Edit" function within our eclipse project is working again, @SebastianBezold would you like to share the onboarding to @arnoweiss or should i overtake?

**Comment by arnoweiss** [comment]
completed
